src/components/nabla/Swap/From.tsx

```diff
@@ -0,0 +1,99 @@
+import { ChevronDownIcon } from '@heroicons/react/20/solid';
+import { Fragment } from 'preact';
```

Comment:
Instead of importing and using Fragment, we might consider using the shorthand `<> </>` syntax for a cleaner approach. What do you all think?